src/Assert.php

```diff
@@ -1401,8 +1460,8 @@ public static function lengthBetween($value, $min, $max, $message = '')
     /**
      * Will also pass if $value is a directory, use Assert::file() instead if you need to be sure it is a file.
      *
-     * @param mixed  $value
-     * @param string $message
+     * @param mixed                    $value
+     * @param string|callable():string $message
```

Comment:
I did not implement it to all assertions. If RFC is accepted, I'll add it everywhere

Reply:
@simPod yes, more verbose. But also more type safe, given that `callable(): string` is not enforceable (without static analysis) but `Stringable|string` is. 🤔

Reply:
In any case, I am not merging new features right now, but this is high priority for the next feature release.

Reply:
Same as `array<object|string>` is not enforceable in `isInstanceOfAny`, that's not valid argument IMO.

_I'm not asking for merge now_